project_task_description_template/tests/test_task_template.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo.tests import common
+
+
+class TestTaskTemplate(common.TransactionCase):
+    def setUp(self):
```

Comment:
You can use `setUpClass` to create every data only one time and not for every test

Reply:
Thanks, I'll fix it